doc-content/enterprise-only/openshift/rolemapping-proc.adoc

```diff
@@ -0,0 +1,68 @@
+[id='rolemapping-proc']
+= Providing the LDAP role mapping file
+
+If you configure the `AUTH_ROLE_MAPPER_ROLES_PROPERTIES` parameter, you must provide a file that defines the role mapping. You must mount this file on all affected deployment configurations.
+
+.Procedure
+
+. Create the role mapping properties file, for example, `my-role-map`. The file must contain lines in the following format:
```

Comment:
Suggestion:
Instead of  "The file must contain lines in the following format",
"The file must contain entries in the following format"